org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/healthcheck/RemoteHealthChecker.java

```diff
@@ -66,4 +69,8 @@ protected boolean execCheckHealth() throws Exception {
     return status;
   }
 
+  @Override
+  public boolean acceptCategory(HealthCheckCategoryId category) {
+    return category == Categories.READINESS;
```

Comment:
equals instead of ==

Reply:
Fixed in 184fa26d2119186562032245b73ffa38f35d5b9e